compiler/control/OMROptions.cpp

```diff
@@ -2138,6 +2138,8 @@ TR::OptionTable OMR::Options::_jitOptions[] = {
     { "x86HLE", "C\tEnable haswell hardware lock elision", SET_OPTION_BIT(TR_X86HLE), "F" },
     { "x86UseMFENCE", "M\tEnable to use mfence to handle volatile store", SET_OPTION_BIT(TR_X86UseMFENCE), "F",
      NOT_IN_SUBSET },
+    { "transientCodeRegex=", "O{regex}\ta regex to specify which classes are transient vs presistent",
```

Comment:
Options need to be ordered alphabetically because the code is doing a binary search on the array of options.
I would change the help message because it's not clear what a transient class is; we came up with this term.
Maybe: "Regex to specify classes that are likely to be unloaded during JVM lifetime. Jitted code for methods belonging to such classes will be stored in dedicated code caches."